src/helpers/configuration_helpers.sh

```diff
@@ -52,6 +69,22 @@ function get_configuration {
 }
 
 
+# Returns whether or not Git Town is configured
+function is_git_town_configured {
+  if [ -n "$main_branch_name" ]; then
```

Comment:
Shouldn't you also check that the non-feature branches are at least set to `''` or something here? What if I forget to provide non-feature branches. Does Git Town think everything is peachey, and just runs?